.github/workflows/bindings.yml

```diff
@@ -162,6 +162,9 @@ jobs:
         with:
           xcode-version: '16.4.0'
       - name: Install swiftlint
+        env:
+          HOMEBREW_NO_AUTO_UPDATE: 1
+          HOMEBREW_NO_INSTALL_UPGRADE: 1
```

Comment:
Is this enough? In the setup rust workflow we do:
```
echo HOMEBREW_NO_AUTO_UPDATE=1 >> $GITHUB_ENV
```

Reply:
Yeah that should do the same, I think in the composite action env context is not available?!